### ISSUE TITLE
build error with latest zig

### ISSUE THREAD

**Issue by wilsonianb** [coilhq/tigerbeetle-node#12]
```
$ yarn install:zig
Installing Zig latest build...
Downloading https://ziglang.org/builds/zig-linux-x86_64-0.9.0-dev.256+0134cb021.tar.xz...
zig-linux-x86_64-0.9.0-dev.256+0134cb021.tar.xz                      100%[====================================================================================================================================================================>]  39.77M  25.8MB/s    in 1.5s    
Extracting zig-linux-x86_64-0.9.0-dev.256+0134cb021.tar.xz...
Installing zig-linux-x86_64-0.9.0-dev.256+0134cb021 to 'zig' in current working directory...
Congratulations, you have successfully installed Zig 0.9.0-dev.256+0134cb021 to /home/brandon/tigerbeetle-node/zig/zig. Enjoy!
```

```
$ yarn build
./src/translate.zig:12:78: error: incompatible types: 'c_uint' and '(enum literal)'
    if (c.napi_create_function(env, null, 0, function, null, &napi_function) != .napi_ok) {
                                                                             ^
./src/node.zig:29:32: note: called from here
    translate.register_function(env, exports, "init", init) catch return null;
                               ^
./src/node.zig:28:88: note: called from here
export fn napi_register_module_v1(env: c.napi_env, exports: c.napi_value) c.napi_value {
                                                                                       ^
./src/translate.zig:12:31: note: type 'c_uint' here
    if (c.napi_create_function(env, null, 0, function, null, &napi_function) != .napi_ok) {
                              ^
./src/translate.zig:12:81: note: type '(enum literal)' here
    if (c.napi_create_function(env, null, 0, function, null, &napi_function) != .napi_ok) {
                                                                                ^
./src/node.zig:29:61: note: referenced here
    translate.register_function(env, exports, "init", init) catch return null;
                                                            ^
./src/node.zig:295:65: error: incompatible types: 'c_uint' and '(enum literal)'
    if (c.napi_get_cb_info(env, info, &argc, &argv, null, null) != .napi_ok) {
                                                                ^
./src/node.zig:295:27: note: type 'c_uint' here
    if (c.napi_get_cb_info(env, info, &argc, &argv, null, null) != .napi_ok) {
                          ^
./src/node.zig:295:68: note: type '(enum literal)' here
    if (c.napi_get_cb_info(env, info, &argc, &argv, null, null) != .napi_ok) {
```

**Comment by ifreund** [comment]
Hey @wilsonianb, thanks for reporting this. I've pushed 5452991a1907c6d0d02d031578cfcc0266e5c39f which pins the zig version used to 0.8.0, which should avoid this happening again as zig continues to make breaking changes in the 0.9.0 release cycle.

**Comment by wilsonianb** [comment]
Thanks, that fixes it locally.
Once a new version is published to npm, I should be able to [install as a dependency](https://github.com/wilsonianb/rafiki/pull/1/checks?check_run_id=2897556577) successfully.

**Comment by ifreund** [comment]
>  Once a new version is published to npm, I should be able to [install as a dependency](https://github.com/wilsonianb/rafiki/pull/1/checks?check_run_id=2897556577) successfully.

Don published version 0.3.2 to npm a few hours ago, this should work now :)
